edge-modules/DirectMethodSender/Settings.cs

```diff
@@ -25,22 +26,38 @@ public class Settings
                     configuration.GetValue<string>("TargetModuleId", "DirectMethodReceiver"),
                     configuration.GetValue<TransportType>("TransportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue<TimeSpan>("DirectMethodDelay", TimeSpan.FromSeconds(5)),
-                    Option.Maybe(configuration.GetValue<Uri>("AnalyzerUrl")));
+                    Option.Maybe(configuration.GetValue<Uri>("AnalyzerUrl")),
```

Comment:
configuration.GetValue<Uri> will throw exception if AnalyzerUrl is defined as empty string.  is this value only defined with non-empty value or not defined?

Reply:
Not defined.